### DISCUSSION TITLE
Version candidate 3.5.1 RC1

### DISCUSSION THREAD

**Discussion by ynaessens** [JeromeDevome/GRR#232] [General]
Corrige plusieurs erreurs de la RC0, notamment l'authentification par SSO ou LDAP.
Veuillez voir le CHANGELOG pour les détails.
Sans retours négatifs de votre part, passera en stable d'ici à trois semaines.

<hr /><em>This discussion was created from the release <a href='https://github.com/JeromeDevome/GRR/releases/tag/v3.5.1RC1'>Version candidate 3.5.1 RC1</a>.</em>

**Comment by acdct** [comment]
Bonjour,

Auriez-vous une prévision de date pour la publication de la version 3.5.1 stable ?

Je suis en train de faire des mises à jour et écrire un rapport pour ma hiérarchie. Je ne sais pas s'il faut que j'attende la version 3.5.1 (si elle sera publiée assez rapidement) ou conclure sur la 3.5.0.

Merci d'avance !

Cordialement

**Reply by ynaessens** [reply]
Bonjour,
j'ai un problème avec les mails automatiques à régler avant la sortie de la 3.5.1 stable. Ceci dit, je pense qu'on a le même avec la 3.5.0.
Si vous êtes prêt à suivre les mises à jour de la 3.5.1, vous gagnerez un cryptage plus robuste des mots de passe dans la base de GRR. Pour moi la RC est "stable", en particulier la structure de la base de données ne va pas encore évoluer, il y a quelques points qu'il serait mieux de régler avant la publication officielle.
Cordialement,
YN

**Reply by acdct** [reply]
Bonjour,
Merci beaucoup pour votre retour.
Je vais donc mettre en place la 3.5.1RC1 puisqu'elle est quasiment stable.
Bonne journée.
Cordialement

**Comment by acdct** [comment]
Re-bonjour,

Je viens d'installer la 3.5.1RC1 et...je n'arrive plus à me connecter avec le compte admin. Y a-t-il quelque chose à faire ?

Merci pour votre aide.

Cordialement

**Reply by ynaessens** [reply]
Bonjour 
Avez-vous fait une installation depuis zéro ou une montée de version ?
Dans le cas d’une montée de version, on constate un refus de connexion à la première tentative mais la deuxième aboutit, pour un administrateur en compte local.
En fresh install, ça devrait passer du premier coup.
Cordialement 
YN

**Reply by acdct** [reply]
Bonjour,
Il s'agit d'une montée de version. J'étais à la 3.5.0b avant.
J'ai supprimé tous les fichiers et remplacé par la version 3.5.1RC1.
La mise à jour de la base de donnée s'est fait avec succès, mais impossible de me connecter même après plusieurs tentatives malheureusement.
J'ai aussi procédé au correctif du typage sur la page language, problème que j'avais en 3.5.0b avec PHP 5.6.40.
Merci

**Reply by acdct** [reply]
Je suis allé vérifier en base, le champ mot de passe n'a pas changé et est resté le même.

Vu que le mode a changé, ce champ n'aurait-il pas dû être modifié lors de la mise à jour BDD en mettant le nouveau cryptage ?

Merci

**Reply by ynaessens** [reply]
Non, en principe le mot de passe se met à jour lors de la première tentative de connexion, pas lors de la mise à jour de la base (si tel était le cas, cela signifierait que la base ancienne version peut être facilement décryptée).
Êtes-vous sûr de votre ancien mot de passe ?
Pouvez-vous rentrer un mot de passe directement en base, crypté par md5 ?

**Reply by acdct** [reply]
Au temps pour moi...cette version prend en compte les majuscules du login, je n'avais pas fait attention.

J'ai réussi à me connecter au bout de la seconde fois comme vous l'aviez indiqué. Désolé !

Encore merci pour votre aide.

**Comment by zpaul91** [comment]
Bonjour, 
La 3.5.1-RC1 est elle suffisamment stable, en dehors des problèmes sécurité (changement de mot de passe depuis admin) et mails auto ? 
Ou avez vous eu d'autres retours négatifs ? 
J'envisage un passage en production avec la 3.5.1 prochainement.
Merci d'avance :-)

**Comment by ynaessens** [comment]
Bonjour,
je suis désolé, je manque de temps actuellement pour finaliser les solutions aux problèmes soulevés, à savoir les mails automatiques et le passage en POST du formulaire de modification de mot de passe.
À ces soucis près, je n'ai pas d'autre point à corriger avant le passage en stable. Sauf si d'autres utilisateurs-testeurs déterrent d'autres problèmes...
Par comparaison, la version 3.5.1, bien que présentée en RC, est la plus aboutie.
Cordialement,
YN

**Reply by zpaul91** [reply]
Merci ! 
Je mergerai sur la version stable quand elle sera dispo, et je pars sur ce qui est actuellement mis.
Bonne journée
